components/Players/PlayerManager.vue

```diff
@@ -388,6 +393,82 @@ export default Vue.extend({
         }
       }
     },
+    addMediaHandlers(): void {
+      if (navigator.mediaSession) {
+        const actionHandlers = [
+          [
+            'play',
+            async (): Promise<void> => {
+              await this.play();
+              if (navigator.mediaSession) {
+                navigator.mediaSession.playbackState = 'playing';
+              }
+            }
+          ],
```

Comment:
```suggestion
            'play',
            (): void => {
              this.unpause();
              if (navigator.mediaSession) {
                navigator.mediaSession.playbackState = 'playing';
              }
            }
          ],
```

I think this is what you want instead

Reply:
Ah yeah I get confused with `this.play` vs `this.unpause`